### PR TITLE
Update module READMEs

### PR DESCRIPTION
**Pull request by lukellmann**
This reduces duplication in the README files of the individual modules and fixes some outdated links and code.

### CODE REVIEW

**Comment by lukellmann** [comment]
also so i don't have to update all these readmes in #961

File: gateway/README.md [hunk 1]
```diff
@@ -11,7 +11,9 @@ suspend fun main(args: Array<String>) {
     val gateway = DefaultGateway { // optional builder for custom configuration
         client = HttpClient {
             install(WebSockets)
-            install(JsonFeature)
+            install(ContentNegotiation) {
+                json()
+            }
```
Comment (DRSchlaubi) [inline gateway/README.md:16]: I am quite sure this is unnecessary

**Review by DRSchlaubi — CHANGES_REQUESTED** [review]

File: gateway/README.md (hunk 1 above)
Comment (lukellmann) [reply gateway/README.md:16]: what part specifically? all of it?

Comment (lukellmann) [reply gateway/README.md:16]: or do you mean because it is gateway and we don't do http requests?

Comment (DRSchlaubi) [reply gateway/README.md:16]: we neither do http requests nor do we use the content negotiation websocket feature

**Review by DRSchlaubi — APPROVED** [review]

**Pull request merged**